### PR TITLE
fix(web-styles): adds styles for form-range in forced-color/high-contrast mode

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: .changeset/great-parrots-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/web-styles': patch
+---
+
+Moves /.stylelintrc.js to /web-styles/.stylelintrc, so it works correctly with the IDE
```
Comment (gfellerph) [inline .changeset/great-parrots-knock.md:5]: A change like this is irrelevant to consumers of our packages, therefore I'd not add it to the changelog. Can you delete this changeset?

File: .changeset/chilled-seahorses-beg.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/web-styles': patch
+---
+
+add styles for form-range in fourced-color/high-contrast mode
```
Comment (gfellerph) [inline .changeset/chilled-seahorses-beg.md:5]:
```suggestion
Add styles for form-range in fourced-color/high-contrast mode
```

File: packages/web-styles/src/components/form-range.scss [hunk 3]
```diff
@@ -43,3 +44,46 @@
...
   }
 }
+
+@include utilities.high-contrast-mode() {
+  .form-range {
+    // linear-gradient is forced to the value of "none" in forces-color mode
+    // so, the "forced-color-adjust" property is necessary for "linear-gradient" to continue to work
+    forced-color-adjust: none;
+
+    &:not(:disabled):not(.disabled) {
```
Comment (gfellerph) [inline packages/web-styles/src/components/form-range.scss:54]:
```suggestion
    &:not(:disabled, .disabled) {
```
The :not pseudo selector accepts lists now 🎉 https://caniuse.com/css-not-sel-list

File: packages/web-styles/src/components/form-range.scss [hunk 4]
```diff
@@ -43,3 +44,46 @@
...
+
+      &::-moz-range-progress {
+        background: ButtonBorder;
+      }
+
+      &::-moz-range-thumb {
+        background-color: ButtonFace;
+        border-color: ButtonBorder;
+      }
+    }
```
Comment (gfellerph) [inline packages/web-styles/src/components/form-range.scss:74]: Can you add :focus-visible styles for HCM? At the moment I can't see my keyboard focus on the slider. Either an outline on the whole slider or just the thumb, but more pronounced than the shadow in non-HCM.

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/web-styles/src/components/form-range.scss (hunk 3 above)
Comment (oliverschuerch) [reply packages/web-styles/src/components/form-range.scss:54]: I just copied the existing selector ;)
So I will change both of them.

**Review by gfellerph — APPROVED** [review]
Good stuff, thank you!

**Review by gfellerph — APPROVED** [review]

**Pull request merged**